client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningInfo.tsx

```diff
@@ -46,7 +46,7 @@ const ContactQuestioningInfo: React.FC<Props> = (props: Props): JSX.Element => {
                 confirmButtonText: 'כן, המשך'
             }).then((result) => {
                 if (result.value) {
-                    if (checkForDuplicateIds(interactedContact.identificationNumber, interactedContact.id) === -1) {
+                    if (!interactedContact.identificationNumber || checkForDuplicateIds(interactedContact.identificationNumber, interactedContact.id) === -1) {
```

Comment:
Why check duplicate ids returns a number?

Reply:
It's an existing function. I just fixed the condition to save null ids.

Reply:
But you're right. It should be fixed to return a boolean value.

Reply:
it returns the result of findIndex